src/Runner.Worker/Container/ContainerHooks/ContainerHookManager.cs

```diff
@@ -194,14 +194,34 @@ private async Task<HookResponse> ExecuteHookScript(IExecutionContext context, Ho
             {
                 throw new Exception($"The hook script at '{HookIndexPath}' running command '{input.Command}' did not execute successfully."); // TODO: better exception
             }
-            Trace.Info($"The hook script at '{HookIndexPath}' running command '{input.Command}' executed successfully.");
+            var response = GetResponse(input);
+            return response;
+        }
+
+        private string GenerateResponsePath()
+        {
+            return Path.Combine(HostContext.GetDirectory(WellKnownDirectory.Temp), ResponseFolderName, $"{Guid.NewGuid()}.json");
+        }
+
+        private string GetPrependPath(IExecutionContext context)
+        {
+            return string.Join(Path.PathSeparator.ToString(), context.Global.PrependPath.Reverse<string>());;
+        }
 
+        private HookResponse GetResponse(HookInput input)
+        {
             HookResponse response = null;
             if (!string.IsNullOrEmpty(input.ResponseFile) && File.Exists(input.ResponseFile))
```

Comment:
Because of `File.Exists(input.ResponseFile))`, we will only throw below if there is a file. We don't throw if there isn't a file for PrepareJob, but I think we should: it can't have isAlpine if the file doesn't exist. We should probably fail earlier.